packages/bus/src/Factories/RMQConnectionFacory.ts

```diff
@@ -0,0 +1,8 @@
+import { ConnectionFactory } from './ConnectionFactory';
+import { connect, Connection } from 'amqplib';
+
+export class RMQConnectionFacory implements ConnectionFactory<Connection> {
+  public async create(url: string, options?: any): Promise<Connection> {
```

Comment:
`async` statement is redundant here

```suggestion
  public create(url: string, options?: any): Promise<Connection> {
```